protos/netstate/netstate.proto

```diff
@@ -51,6 +54,7 @@ message ListResponse {
 
 message DeleteRequest {
   bytes path = 1;
+  bytes xApiKey = 2;
```

Comment:
the x is to assume it'd be part of the header or something the client is submitting.  I'll leave it for now, as you suggested.

Reply:
heh, so, i do think we should "drop the x... it's cleaner", but i think we should leave it a field and not use a grpc header and leave it as is, is what i meant to say. don't worry about making this a grpc header.

this is a great opportunity for two of my favorite tools! `gofmt -r` is a rewrite rule system. you can automatically rename a variable in your entire codebase. here's a good blog post: https://fallthrough.io/2015/08/gofmt-rewrite-rules/

more simply is `sed -i`. you can do a regex search and replace. so like, `find -type f -name "*.go" -print0 | xargs -0 sed -i 's/xApiKey/ApiKey/g'`. if you don't know about find and xargs (maybe you do!, but for our readers at home: https://twitter.com/b0rk/status/992636366075973633 is good). sed is here: https://pbs.twimg.com/media/DeN10NLUQAApvNY.jpg:large

the reason i bring this up is one other change, which is that Go naming convention is such that acronyms should all be capitalized. APIKey instead of ApiKey, but apiKey is okay.

i realize this one request (change the capitalization of this field) is super obnoxious, but probably easier to fix now than in a few months!

Reply:
no worries!